### DISCUSSION TITLE
Dynamic skipped files

### DISCUSSION THREAD

**Discussion by tpluscode** [copier-org/copier#604] [Q&A]
Is it possible to use jinja template to dynamically set files to skip?

Like

```
_skip_if_exists:
{%- for api in apis %}
  - ".copier-{{api}}.yaml"
{%- endfor %}
```

**Comment by yajo** [comment]
You cannot template outside values, so the loop is not possible.

In the case of the `{{api}}` part, TBH I can't remember if this is valid. Have you tried?

**Reply by tpluscode** [reply]
> You cannot template outside values, so the loop is not possible.

what does that mean exactly?

**Reply by pawamoy** [reply]
I believe what @Yajo wanted to say is that, the individual YAML *values* are rendered, not the *whole file*. The file *must* be valid YAML. Your example is not :slightly_smiling_face:
